src/icons/src/svg/arrow-xsmall.svg

```diff
@@ -1 +1,3 @@
-<svg width="14" height="9" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M5.984 7.578L.424 1.514 1.9.163l5.22 5.695L12.341.163l1.475 1.351-5.56 6.064a1.542 1.542 0 01-2.272 0zm1.474-1.351z"/></svg>
+<svg width="8" height="12" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
hmmm why as this changed?

Reply:
I updated it—designs have changed. Team's moving over to Google's Material icons for the utility icons (arrows, downloads, etc.)... but frankly that isn't all organized yet (and won't be until I do it) so I can rip out this change from this ticket for now. (I have, however, already pushed up the change for arrows in Reno, so I'd like to do it here if I can.)

Reply:
sounds good